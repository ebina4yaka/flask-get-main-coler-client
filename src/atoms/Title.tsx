import { ReactElement } from 'react'
import Typography from '@material-ui/core/Typography'

export default function Title(): ReactElement {
  return (
    <Typography variant="h4" component="h1" gutterBottom>
      Get main color from your image
    </Typography>
  )
}
