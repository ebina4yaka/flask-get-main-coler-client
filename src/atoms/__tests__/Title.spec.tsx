import renderer from 'react-test-renderer'
import Title from '../Title'

test('Snapshot test for Title', () => {
  const tree = renderer.create(<Title />).toJSON()
  expect(tree).toMatchSnapshot()
})
