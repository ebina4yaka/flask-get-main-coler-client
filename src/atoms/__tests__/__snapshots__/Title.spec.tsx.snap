// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for Title 1`] = `
<h1
  className="MuiTypography-root MuiTypography-h4 MuiTypography-gutterBottom"
>
  Get main color from your image
</h1>
`;
